app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,15 @@
 class ApplicationMailer < ActionMailer::Base
   default from: "from@example.com"
   layout "mailer"
+
+  around_action :with_organization_path, if: -> { Current.organization }
+
+  private
+
+  def with_organization_path
+    default_url_options[:script_name] = "/#{Current.organization.slug}"
+    yield
+  ensure
+    default_url_options[:script_name] = nil
+  end
```

Comment:
I'm worried about the thread safety of this code but we don't have a better solution so I guess it's better than nothing :)

Reply:
Yes let's go with it until we need to do something else...if that comes.